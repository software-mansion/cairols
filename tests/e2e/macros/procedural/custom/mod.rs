mod v1;
mod v1_and_v2;
mod v2;
