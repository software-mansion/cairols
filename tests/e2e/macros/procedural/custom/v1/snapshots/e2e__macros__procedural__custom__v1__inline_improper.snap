---
source: tests/e2e/macros/procedural/custom/mod.rs
expression: report
---
[[expansions]]
analyzed_lines = """
    let _x = improper<caret>_inline_macro!(10 + 10);
    let _x = improper_inline_macro!(10 +<caret> 10);
"""
generated_code = """
// lib.cairo
// ---------

unbound_identifier
"""

mapped_diagnostics:
- url: '[ROOT_URL]test_package/src/lib.cairo'
  diagnostics:
  - related_code: improper_inline_macro!(10 + 10)
    diagnostic:
      range:
        start:
          line: 1
          character: 13
        end:
          line: 1
          character: 44
      severity: 1
      code: E0006
      message: Identifier not found.
      relatedInformation:
      - location:
          uri: vfs://inline_proc_macro.cairo
          range:
            start:
              line: 0
              character: 0
            end:
              line: 0
              character: 18
        message: Diagnostic mapped from here.
